rviz_visual_testing_framework/include/rviz_visual_testing_framework/visual_test_fixture.hpp

```diff
@@ -46,14 +46,23 @@
 #include "rviz_visual_testing_framework/internal/visual_test.hpp"
 #include "rviz_visual_testing_framework/page_objects/page_object_with_window.hpp"
 
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <ament_index_cpp/get_package_prefix.hpp>
+
 class VisualTestFixture : public testing::Test
 {
 public:
   VisualTestFixture()
   {
     test_name_ = ::testing::UnitTest::GetInstance()->current_test_info()->name();
     executor_ = std::make_shared<Executor>();
-    src_directory_path_ = std::string(_SRC_DIR_PATH);
+    //src_directory_path_ = std::string(_SRC_DIR_PATH);
+    std::string ROS_PACKAGE_NAME = "rviz_visual_testing_framework";
+    try {
+      src_directory_path_ = ament_index_cpp::get_package_share_directory(ROS_PACKAGE_NAME);
+    }   catch(ament_index_cpp::PackageNotFoundError& e) {
+      std::cerr<<"Caught PackageNotFoundError for package name '"<<e.package_name;
```

Comment:
This violates our style (need space between operators like `<<` and other things), please run the tests locally to clear up these kinds of things. 👍

Reply:
Noted. Will look into the code style and clang tools available.

Reply:
Ran the following and made changes as necessary
    ament_cpplint
    ament_uncrustify